packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte

```diff
@@ -10,7 +10,7 @@
         NftDownloadMetadata,
         allWalletNfts,
         convertAndFormatNftMetadata,
-        getNftByIdFromAllAccountNfts,
```

Comment:
`getNftByIdFromAllAccountNfts` still exists in `packages/shared/components/activity-info/NftActivityInformation.svelte` && `packages/shared/components/activity-info/NftMetadataInformation.svelte`